drivers/wifi/nrf700x/CMakeLists.txt

```diff
@@ -136,6 +136,13 @@ zephyr_compile_definitions_ifdef(CONFIG_NRF700X_ON_QSPI
   -DNRF52_ERRATA_215_ENABLE_WORKAROUND=0
 )
 
+# QSPI doesn't use 192MHz clock and most samples use 128MHz, this can cause Anamoly 159
+# but as its rare and not seen in most cases, we can disable it.
+# Alternative is 128MHz CPU should be disabled that impacts Wi-Fi performance.
+zephyr_compile_definitions(
+  -DNRF53_ERRATA_159_ENABLE_WORKAROUND=0
```

Comment:
I think this should be disabled only if QSPI is used for communication with nRF700x, so as above, `zephyr_compile_definitions_ifdef(CONFIG_NRF700X_ON_QSPI` should be used here.